### DISCUSSION TITLE
Live previews on desktop or mobile

### DISCUSSION THREAD

**Discussion by froyo49** [webstudio-is/webstudio-community#66] [Ideas]
It'd be great to see a live preview of my site on my mobile device or on my computer outside of the editor. The share link just links to the editor, even when only the "view" checkbox is selected. Since all iOS browsers are WebKit based, it doesn't work. The only solution is to publish the whole site when I make a change, which is really time consuming. 

Seeing the site on a real mobile device or browser window gives me a better understanding of what it'll look like in the real world. For example, since people tend to hold their phones closer to their head than desktops, I can figure out the right size for text or other elements if I physically see it on my phone. This would be an immensely valuable feature. 

Thank you!

**Comment by kof** [comment]
100% agree

**Comment by cpiprint** [comment]
Yup not working on mobile browsers....

**Reply by kof** [reply]
builder is not intended to work on mobile browsers, but the canvas will be launchable from any device in the future
